algebird-test/src/test/scala/com/twitter/algebird/CountMinSketchTest.scala

```diff
@@ -86,6 +85,11 @@ class TopPctCmsLaws extends PropSpec with PropertyChecks with Matchers {
   val SEED = 1
   val HEAVY_HITTERS_PCT = 0.1
 
+  implicit def topCmsEquiv[K]: Equiv[TopCMS[K]] =
```

Comment:
Ugh, annoyingly that fails.  The problem is when you compare `++` and `sumOption` on a singleton list.